### ISSUE TITLE
[Feature] How to modify UI based on `CardComponent`?

### ISSUE THREAD

**Issue by yujingning** [Adyen/adyen-ios#766]
**Describe the challenge**
Hi~
I'm using `CardComponent`,  but our design is different from the current UI provided by `CardComponent`. Is there a way to make some modifications based on `CardComponent`? Such as add a `UILabel` above the confirm button.

![image](https://user-images.githubusercontent.com/81403461/150100213-0534285c-53be-41ea-923e-bf92592c8bab.png)

SDK Version: 4.4.0

**Comment by yujingning** [comment]
Find the corresponding `stackview` through the view hierarchy, and then insert a custom` view` into the `stackView`. 
```    
   guard let componentViewController = cardComponent?.viewController as? SecuredViewController,
              let rootViewController = componentViewController.children.first as? FormViewController,
              let scrollView = rootViewController.view.subviews.first as? UIScrollView,
              let stackView = scrollView.subviews.first as? UIStackView else {
            return
        }
  guard stackView.subviews.count > 4 else { return }
  stackView.insertArrangedSubview(tipStackView, at: 3)
```

**Comment by descorp** [comment]
Hey @yujingning 

There is no easy way to do it, but we are planing to decouple form and "Submit" button in v5.

**Comment by yujingning** [comment]
> Hey @yujingning
> 
> There is no easy way to do it, but we are planing to decouple form and "Submit" button in v5.

Okay～Thanks for your quick reply. If we can control the Views on `CardComponent`, it will be very convenient for us to make some modifications 
Now I find the corresponding `StackView` in `CardComponent` through the view hierarchy, and then I can insert a custom View.

**Comment by lpuentes19** [comment]
I'm currently using the 5.0.0 Adyen SDK and have this exact same question. We tried following @yujingning steps in casting our way to the stackView, but it looks like SecuredViewController and FormViewController are both internal and not accessible in the code and therefore throws an error. Is there a way to add in a custom view to the Card Component form? 


Also, is it possible to add a border for the textField's and not have it be just a line below the text? @descorp @mohammedDehairy

**Comment by neelSharma12** [comment]
@yujingning We cannot do any modifications on ```CardComponent``` for adding custom views at the moment.  @lpuentes19 Thank you for sharing your feedback. I will share this with concerned team but for now could you please create a separate ticket for adding the border to the textFields?

**Comment by goergisn** [comment]
We just shipped [5.14.0](https://github.com/Adyen/adyen-ios/releases/tag/5.14.0) which allows you to use a custom pay button and thus add content below the form.

I'm closing the issue but feel free to re-open if there are any additional questions. :)
